src/labkey/filter/Types.ts

```diff
@@ -140,18 +140,22 @@ export let Types: Record<string, IFilterType> = {
     //
     // Table/Query-wise operators
     //
-    Q: registerFilterType('Search', null, 'q', true, undefined, 'Search across all columns', undefined, undefined, true)
+    Q: registerFilterType('Search', null, 'q', true, undefined, 'Search across all columns', undefined, undefined, true),
+    //
+    // Ontology operators
+    //
+    ONTOLOGY_IN_SUBTREE : registerFilterType("Is In Subtree", null, "concept:insubtree", true,),
+    ONTOLOGY_NOT_IN_SUBTREE : registerFilterType("Is Not In Subtree", null, "concept:notinsubtree", true,)
```

Comment:
minor: extra comma after the last param in `registerFilterType()` for both of these.